src/templates/resources/footer.json

```diff
@@ -922,7 +922,7 @@
             },
             {
                 "url": "/takeaway/",
-                "text": "View all locations"
+                "text": "Browse more locations"
```

Comment:
In consumer web this line is "Browse more cities". Should we make them consistent?

Reply:
@dandel10n thanks for your review. Updated text message.